### PR TITLE
2.0.2 show more jobs | Physical Person and 2.0.3 Jobs details | Physical Person 

### PR DESCRIPTION
**Pull request by ClaraFelix**

### CODE REVIEW

File: apps/frontend/src/app/pages/jobs-details/jobs-details.component.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        <p class="font-bold text-sm text-base-100 font-mono">Back</p>
+      </div>
+      <div class="pt-6 flex gap-6 pb-8">
+        <div class="flex gap-2">
+          <i class="bi bi-clock text-primary"></i>
+          <p class="font-mono text-base-200">{{ time }}</p>
+        </div>
+        <div class="flex gap-2">
+          <i class="bi bi-laptop text-primary"></i>
+          <p class="font-mono text-base-200">{{ locale }}</p>
```
Comment (dev-luizf) [inline apps/frontend/src/app/pages/jobs-details/jobs-details.component.html:18]: ><p class="font-mono text-base-200">{{ locale }}</p>
should be allocation

File: apps/frontend/src/app/pages/jobs-details/jobs-details.component.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        </div>
+      </div>
+      <div class="pb-8">
+        <h3 class="font-sans font-medium text-2xl text-base-100 pb-2">
+          {{ name }}
```
Comment (dev-luizf) [inline apps/frontend/src/app/pages/jobs-details/jobs-details.component.html:23]: >{{ name }}

should be title

File: apps/frontend/src/app/pages/jobs-details/jobs-details.component.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        </h3>
+        <p class="font-mono text-base-200">{{ responsibilities }}</p>
```
Comment (dev-luizf) [inline apps/frontend/src/app/pages/jobs-details/jobs-details.component.html:25]: Responsabilities and description should be the same element. We won't divide it in different things, it is all description.

File: apps/frontend/src/app/pages/jobs-details/jobs-details.component.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        <p class="font-mono text-base-200">{{ description }}</p>
+      </div>
+      <button class="btn btn-primary w-[200px]">Apply</button>
+    </div>
+    <div class="pt-20">
+      <div
+        class="bg-neutral md:w-[416px] md:h-[336px] w-full h-full rounded-lg p-8"
+      >
+        <div class="flex items-center pb-6">
+          <img
```
Comment (dev-luizf) [inline apps/frontend/src/app/pages/jobs-details/jobs-details.component.html:38]: The image and the company name are too close:

Captura de tela de 2024-03-13 15-13-37

File: apps/frontend/src/app/pages/jobs/jobs.component.html
Comment (dev-luizf) [inline apps/frontend/src/app/pages/jobs/jobs.component.html:1]: 1. Cards cursor should be pointer.
2. The card description should have a overflow with "..." and not cut off halfway through the last line.
Captura de tela de 2024-03-13 15-01-28

3. The search is not working.
Captura de tela de 2024-03-13 15-18-53

4. The "all" filter should be selected by default when the user enters the page.
Captura de tela de 2024-03-13 15-19-23

**Review by dev-luizf — CHANGES_REQUESTED** [review]

**Review by Bidfca — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/shared/components/cards/cards.component.html [hunk 5]
```diff
@@ -1,34 +1,57 @@
...
+          details === 'more'
+            ? ' text-base font-mono font-bold'
+            : 'font-sans font-medium text-2xl'
+        ]"
       >
         {{ name }}
       </h3>
       <div class="pb-6">
-        <p class="text-base-200 font-mono h-[108px] line-clamp-2">
+        <p class="text-base-200 font-mono h-24 overflow-hidden line-clamp-3">
```
Comment (Bidfca) [inline apps/frontend/src/app/shared/components/cards/cards.component.html:49]: Remember to apply these changes in https://github.com/Venture-Miner/wagademy/pull/63 too

File: apps/frontend/src/app/pages/jobs-details/jobs-details.component.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+        class="flex gap-2 items-center py-4 cursor-pointer"
+        [routerLink]="['/pages/jobs']"
+      >
+        <i class="bi bi-arrow-left text-base-100 text-xl"></i>
+        <p class="font-bold text-sm text-base-100 font-mono">Back</p>
+      </div>
+      <div class="pt-6 flex gap-6 pb-8">
+        <div class="flex gap-2">
+          <i class="bi bi-clock text-primary"></i>
+          <p class="font-mono text-base-200">{{ jobType }}</p>
```
[the remark this answers is not in the document]
Comment (ClaraFelix) [reply apps/frontend/src/app/pages/jobs-details/jobs-details.component.html:14]: Bia, I couldn't identify the font problem. Is it no longer working?

image
image

**Pull request merged**